examples/wasi-hello-world/wasi-hello-world.rust.en-us.md

```diff
@@ -0,0 +1,80 @@
+# WASI Hello World
+
+## Overview
+
+In this example, We will be writing "Hello world!" to both the console (`/dev/stdout`), and a newly created file `./hello-world.txt`. We highly reccomended that you have read the [WASI Introduction](/example-redirect?exampleName=wasi-introduction) before procedding with this example. You should install [wasmtime](https://wasmtime.dev/) as that is the WebAssembly runtime we will be using as our host. You should also ensure that [rustup and rust](https://www.rust-lang.org/tools/install) is installed.
```

Comment:
`stdout` is just the name of a POSIX io stream (see `man 3 stdout`), the /dev/stdout file is a special Linux thing.

Reply:
"you should install Rust using the [`rustup`](url here) tool"